### PR TITLE
Numerics

### PR DESCRIPTION
**Pull request by stevehoag**
Ported from MSDN

### CODE REVIEW

File: docs/essentials/numericsincore.md [hunk 1]
```diff
@@ -1 +1,58 @@
+# Numerics in .NET Core
 
+.NET Core supports the standard numeric integral and floating-point primitives, as well as [System.Numerics.BigInteger](http://dotnet.github.io/api/System.Numerics.BigInteger.html), an integral type with no theoretical upper or lower bound, [System.Numerics.Complex](http://dotnet.github.io/api/System.Numerics.Complex.html), a type that represents complex numbers, and a set of Single Instruction Multiple Data (SIMD)-enabled vector types in the [System.Numerics](http://dotnet.github.io/api/System.Numerics.html) namespace.
```
Comment (svick) [inline docs/essentials/numericsincore.md]: Would it make sense to link SIMD to [its Wikipedia page](https://en.wikipedia.org/wiki/SIMD)?

File: docs/essentials/numericsincore.md [hunk 2]
```diff
@@ -1 +1,58 @@
...
+
+The **Double** and **Single** types are intended to be used for values that by their nature are imprecise (such as the distance between two stars in the solar system) and for applications in which a high degree of precision and small rounding error is not required. You should use the **Decimal.GetBits** type for cases in which greater precision is required and rounding error is undesirable.
```
Comment (svick) [inline docs/essentials/numericsincore.md]: >  You should use the **Decimal.GetBits** type

This is copied from MSDN, but I think it's supposed to be "the **Decimal** type".

Comment (svick) [inline docs/essentials/numericsincore.md]: Also, I think this does not explain the difference between decimal and binary floating point types well. It doesn't even mention decimal fractions.

File: docs/essentials/numericsincore.md (hunk 1 above)
Comment (stevehoag) [inline docs/essentials/numericsincore.md]: @svick I don't know what our policy is - do we link to Wikipedia from any other topics?

File: docs/essentials/numericsincore.md (hunk 2 above)
Comment (stevehoag) [inline docs/essentials/numericsincore.md]: @svick Can you suggest a better explanation? What else needs to be said?

File: docs/essentials/numericsincore.md (hunk 1 above)
Comment (richlander) [inline docs/essentials/numericsincore.md]: Totally fine to link to wikipedia.

Comment (svick) [inline docs/essentials/numericsincore.md]: @stevehoag According to `git grep`, there are 10 links to Wikipedia already.

File: docs/essentials/numericsincore.md (hunk 2 above)
Comment (svick) [inline docs/essentials/numericsincore.md]: Something like:

> The **Double** and **Single** types are intended to be used for values that by their nature are imprecise (such as the distance between two stars in the solar system) and for applications in which a high degree of precision is not required and small rounding error is acceptable. The **Decimal** type can accurately represent [decimal fractions](https://en.wikipedia.org/wiki/Decimal_fraction) (like monetary values) and it also generally has greater precision than the other floating-point types.

Though the claim that `decimal` can accurately represent any decimal fraction in incorrect (it's still limited by the size of its mantissa), I think it's a reasonable simplification for this document.

**Comment by BillWagner** [comment]
Read through this today. LGTM.  @stevehoag should I go ahead and merge this?

**Pull request merged**